examples/4_sgcnn/sgcnn.jl

```diff
@@ -10,18 +10,17 @@ using Random
 using ChemistryFeaturization
 using AtomicGraphNets
 using Serialization
-<<<<<<< HEAD
+<< << << < HEAD
```

Comment:
Should this be here? Looks like this was overlooked while resolving a merge conflict...

Reply:
haha yuuup thought I'd fixed that, oops